todo_app/data/trello_items.py

```diff
@@ -0,0 +1,164 @@
+from requests import request
+from os import environ
+from todo_app.data.Item import Item
+
+TRELLO_KEY = environ["TRELLO_KEY"]
+TRELLO_TOKEN = environ["TRELLO_TOKEN"]
+TRELLO_BOARD_ID = environ["TRELLO_BOARD_ID"]
+
+list_ids={}
+
+def update_from_trello():
+    """
+    Fetches all ticket information from Trello
+    Returns:
+        dict: json response from trello
+    """
+    url= f"https://api.trello.com/1/boards/{TRELLO_BOARD_ID}/lists"
+
+    headers = {
+        "Accept": "application/json"
+    }
+
+    query = {
+        "key":TRELLO_KEY,
+        "token":TRELLO_TOKEN,
+        "cards":"open"
+        }
+
+    response = request(
+        "GET",
+        url, 
+        headers=headers, 
+        params=query, 
+        verify=False # This is just to get past the corporate proxy I am developing on. I understand not to use this flag if this were production software.
+    ).json()
+  
+    return response
+
+
+def parse_trello_response(response):
+    """ Parses trello response into a list of dictionaries with the form 
+    {
+        "title":<CARD_NAME>,
+        "id":<CARD ID_SHORT>
+        "status":<TRELLO_LIST THAT CONTAINS CARD>
+        "trello_id":<TRELLO_ID>
+    }
+
+    Args:
+        List: cards on the trello board
+    """
+    cards = []
+
+    for trello_list in response:
+        list_status = trello_list["name"]
+
+        list_ids[list_status] = trello_list["id"]
+
+        for card in trello_list["cards"]:
+            cards.append(Item.from_trello_card(card, trello_list))
+    
+    return cards
+
+
+def get_items():
+    """
+    Fetches all saved items from trello.
+
+    Returns:
+        list: The list of saved items.
+    """
+    response = update_from_trello()
+    cards = parse_trello_response(response)
+
+    return cards
+
+
+def get_item(id):
```

Comment:
Minor but worth noting that this function is never used, so it could be worth getting rid of rather than maintaining it for no value